app/controllers/nonsipp/landorproperty/LandOrPropertyListController.scala

```diff
@@ -118,6 +118,7 @@ object LandOrPropertyListController {
 
     val title = if (addresses.size == 1) "landOrPropertyList.title" else "landOrPropertyList.title.plural"
     val heading = if (addresses.size == 1) "landOrPropertyList.heading" else "landOrPropertyList.heading.plural"
```

Comment:
Just to double-check, is there any other journey (e.g. shares, other assets, etc) where the same UI logic should be applied and it is not already there?